projects/opendr_ws/src/perception/scripts/video_activity_recognition.py

```diff
@@ -31,33 +31,40 @@
 
 
 class HumanActivityRecognitionNode:
-
     def __init__(
         self,
-        input_image_topic="/usb_cam/image_raw",
+        input_rgb_image_topic="/usb_cam/image_raw",
         output_category_topic="/opendr/human_activity_recognition",
         output_category_description_topic="/opendr/human_activity_recognition_description",
         device="cuda",
-        model='cox3d-m'
+        model="cox3d-m",
     ):
         """
         Creates a ROS Node for face recognition
```

Comment:
This description should probably be updated, i am adding a basic suggestion, feel free to change.

```suggestion
        Creates a ROS Node for video-based human activity recognition.
```